pkg/framework/aries/framework.go

```diff
@@ -220,8 +224,11 @@ func createWallet(frameworkOpts *Aries, ctxProvider api.Provider) error {
 	return nil
 }
 
-func createOutboundDispatcher(frameworkOpts *Aries, ctxProvider dispatcher.Provider) error {
-	var err error
+func createOutboundDispatcher(frameworkOpts *Aries) error {
+	ctxProvider, err := frameworkOpts.Context()
```

Comment:
why create the context twice instead of passing in from parent?

Reply:
1- Wallet need context 
2- OutboundDispatcher need context that has wallet
3- Services need context that has OutboundDispatcher

Reply:
@troyronda we create issue to address this issue

Reply:
It seems like we shouldn't be reusing the same Context() method for everything. The first two seem like special bootstrap steps.

(Re-using Context() for bootstrap is confusing - rather there should be specialized private methods that return only the subset type).

Reply:
@troyronda we will fix in  https://github.com/hyperledger/aries-framework-go/issues/212

Reply:
@troyronda I change it